torch/_dynamo/utils.py

```diff
@@ -1364,17 +1364,24 @@ def get_fake_value(node, tx):
     if "example_value" in node.meta and is_fake(node.meta["example_value"]):
         return node.meta["example_value"]
 
-    def fake_wrapper(e):
+    def ensure_fake(e):
         if isinstance(e, torch.Tensor):
             assert is_fake(e)
```

Comment:
Should we also ensure that the fake of the args is this graph's fake?

Reply:
I think the answer is yes - as long as we can get away with it, stricter is better unless someone has a good reason to make an exception.

Reply:
Let's monitor if anything goes wrong as a result